get_ecg.py

```diff
@@ -0,0 +1,173 @@
+import numpy as np
+import math
+
+
+class GetEcg:
```

Comment:
nit: Generally _functions_ (not classes) should start with a verb like you have here, because they will go do something (e.g. `Get` stuff or `Calculate` stuff, etc) but generally classes tend to be named as entities or objects (e.g. `Cat`, `Monitor`, `ECG`, etc).